tests/fdb/test_fdb_mac_learning.py

```diff
@@ -173,10 +169,25 @@ def prepare_test(self, duthosts, rand_one_dut_hostname, ptfhost):
         logging.info("shutdown all interfaces on DUT")
         for port in dut_ports:
             duthost.shell("sudo config interface shutdown {}".format(port))
-        duthost.command('sudo config save -y')
-        config_reload(duthost, config_source='config_db', safe_reload=True)
+
         yield target_ports_to_ptf_mapping, ptf_ports_available_in_topo, conf_facts
 
+        logging.info("startup all interfaces on DUT")
+        for port in dut_ports:
+            duthost.shell("sudo config interface startup {}".format(port))
```

Comment:
What's the purpose for this change? We have some admin down ports that are not connected.

Reply:
this testcase doesn't startup those admin down interfaces in teardown previously, this is to restore those interfaces.